autopush/websocket.py

```diff
@@ -986,6 +1000,9 @@ def finish_webpush_notifications(self, result):
             self.ps.updates_sent[str(notif.channel_id)].append(notif)
             msg = notif.websocket_format()
             messages_sent = True
+            update_cnt += 1
```

Comment:
Fine doing that, but wouldn't the arg fall out of scope pretty fast once we're out of this function? (I note that pycharm greys out the var as unused once we're past the loop.)